pip/download.py

```diff
@@ -117,6 +118,13 @@ def user_agent():
     )
 
 
+def unquote(s):
+    if six.PY2:
+        return urllib_unquote(s.encode("utf-8")).decode("utf-8")
```

Comment:
Since [internally](https://github.com/python/cpython/blob/2.7/Lib/urllib.py#L1230) `urllib.unquote` will simply re-en/decode these as latin1, I can't see the point of the little utf-8 jig that's written here, but maybe I've thought about it wrong?